src/lib.rs

```diff
@@ -180,3 +227,20 @@ impl Image for Neo4j {
         Box::new(self.env_vars.iter())
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use crate::{Neo4j, Neo4jLabsPlugin};
+
+    #[test]
+    fn single_plugin_definition() {
+        let neo4j = Neo4j::default().with_neo4j_labs_plugin(vec![Neo4jLabsPlugin::Apoc]);
+        assert_eq!(neo4j.env_vars.get("NEO4JLABS_PLUGINS").unwrap(), "[\"apoc\"]");
+    }
+
+    #[test]
+    fn multiple_plugin_definition() {
+        let neo4j = Neo4j::default().with_neo4j_labs_plugin(vec![Neo4jLabsPlugin::Apoc, Neo4jLabsPlugin::Bloom]);
+        assert_eq!(neo4j.env_vars.get("NEO4JLABS_PLUGINS").unwrap(), "[\"apoc\",\"bloom\"]");
+    }
```

Comment:
With the slice in the signature, this now looks like

```suggestion
    fn single_plugin_definition() {
        let neo4j = Neo4j::default().with_neo4j_labs_plugin(&[Neo4jLabsPlugin::Apoc]);
        assert_eq!(neo4j.env_vars.get("NEO4JLABS_PLUGINS").unwrap(), "[\"apoc\"]");
    }

    #[test]
    fn multiple_plugin_definition() {
        let neo4j = Neo4j::default().with_neo4j_labs_plugin(&[Neo4jLabsPlugin::Apoc, Neo4jLabsPlugin::Bloom]);
        assert_eq!(neo4j.env_vars.get("NEO4JLABS_PLUGINS").unwrap(), "[\"apoc\",\"bloom\"]");
    }
```